apps/api/src/app/messages/usecases/get-messages/get-messages.usecase.ts

```diff
@@ -7,7 +7,11 @@ export class GetMessages {
   constructor(private messageRepository: MessageRepository, private subscriberRepository: SubscriberRepository) {}
 
   async execute(command: GetMessagesCommand) {
-    const LIMIT = 10;
+    const LIMIT = command.limit;
+
+    if (LIMIT > 1000) {
```

Comment:
Is this validation needed? Should it be larger or smaller?

Reply:
I am not sure. I tend to think it should be smaller. At least until a need for more comes up